sources/core/Stride.Core.Yaml.Tests/Serialization/SerializationTests.cs

```diff
@@ -816,6 +816,24 @@ public void DeserializeWithRepeatedSubObjects()
             Assert.Equal(22, family.Mother.Age);
         }
 
+        [Fact]
+        public void ThrowWithoutEmptyCtor()
+        {
+            try
+            {
+                SerializeThenDeserialize(new ClassWithNonEmptyCtor(default));
+            }
+            catch (Exception ex)
+            {
+                Assert.True(ex.InnerException is DefaultObjectFactory.InstanceCreationException);
```

Comment:
cant you use here Assert.Throws?

Reply:
I looked at it initially but didn't find an overload for inner exceptions ?

Reply:
` Assert.That(ex.InnerException, Is.TypeOf<BadException>() );` mybe that one?

Reply:
Changed to that one instead `Assert.IsType<DefaultObjectFactory.InstanceCreationException>(ex.InnerException);`, no API for `.That()`